studies/serializers.py

```diff
@@ -1,10 +1,90 @@
-from rest_framework import serializers
+from accounts.models import Child, Organization, User
+from api.serializers import (ModelSerializer, UUIDResourceRelatedField,
+                             UUIDSerializerMixin)
+from rest_framework_json_api import serializers
+from studies.models import Response, Study
 
-from studies.models import Study
 
-
-class StudySerializer(serializers.ModelSerializer):
+class StudySerializer(UUIDSerializerMixin, ModelSerializer):
+    url = serializers.HyperlinkedIdentityField(
+        view_name='study-detail',
+        lookup_field='uuid'
+    )
+    organization = UUIDResourceRelatedField(
+        queryset=Organization.objects,
+        many=False,
```

Comment:
Yeah, there's only one organization per study. It's a FK. There's no documented way to do this in drf-jsonapi, at least not that I can find. It is a list of one. Changing it from a list view to a detail view didn't work because the detail view expects you to pass the UUID in the url, which it doesn't do.